### ISSUE TITLE
momentum in BatchNorm

### ISSUE THREAD

**Issue by Atcold** [pytorch/pytorch#41559]
## 📚 Documentation

What's going on with `momentum` in BatchNorm? `0.9` of momentum means I keep `0.9` of the old stuff and add `0.1` of the new one. Why is it reversed here? `momentum` equal `0` means I'm running with no memory of the past.

**Comment by zou3519** [comment]
I'm not sure why or how we picked the defaults. Labelling this as "needs research" for someone to come and take a pass at it.

**Comment by mruberry** [comment]
The typical momentum term does appear on the historic search direction, based on a brief literature search, but a default momentum value of .9 is common.

**Comment by Atcold** [comment]
@ylecun weights in:

> A running average is a 1st order low-pass filter.
>
> `y <- alpha * x + (1 - alpha) * y`
>
> The cut-off frequency frequency is `fc = alpha / (2 * pi * (1 - alpha))`, the "cut-off period" is `pc = 2 * pi * (1 - alpha) / alpha`. So, for `alpha = 0.1`, `pc = 56.5` and for `alpha = 0.01`, `pc = 622`.
> So for `alpha = 0.1`, you get a lot of variance, the variance you would observe by averaging about `50` samples, which reduces the the standard deviation by `√50`.

**Comment by mruberry** [comment]
It's not clear to me that @ylecun is arguing for one value over another in that statement, though? Also, interesting to note that @ylecun's alpha is on the new, not historic, term.

**Comment by Atcold** [comment]
@mruberry, currently PyTorch is calling `alpha` "momentum" (see the API)!
If we want to stick with the term "momentum" this should be defined as `1 - alpha`.

The one I've posted above was the reply to my question «Why should we go for `alpha = 0.01` instead of `0.1`?».

**Comment by mruberry** [comment]
> @mruberry, currently PyTorch is calling `alpha` "momentum" (see the API)!
> If we want to stick with the term "momentum" this should be defined as `1 - alpha`.
> 
> The one I've posted above was the reply to my question «Why should we go for `alpha = 0.01` instead of `0.1`?».
> Notice the «you get _a lot of variance_» part.

OK. But then why not .001?
